util/settings/settings_test.go

```diff
@@ -805,49 +805,6 @@ func TestSettingsManager_GetEventLabelKeys(t *testing.T) {
 	}
 }
 
-func TestKustomizeSettings_GetOptions(t *testing.T) {
```

Comment:
These tests have been copied and adapted to test the equivalent function, `getKustomizeBinaryPath` in reposerver/reposerver/repository.go.